app/templates/base-flat.jade

```diff
@@ -31,35 +31,44 @@ mixin accountLinks
                 span.icon-bar
                 span.icon-bar
                 span.icon-bar
-              a.navbar-brand(href="/home")
-                if serverConfig.codeNinjas
-                  img#logo-img.code-ninjas-powered-by(src="/images/pages/base/logo.png")
-                  img.code-ninjas-logo(src="/images/pages/base/code-ninjas-logo-right.png")
-                else
-                  img#logo-img(src="/images/pages/base/logo.png")
-                span.glyphicon.glyphicon-home
+              if me.useTarenaLogo()
+                a.navbar-brand(href="http://kidtts.tmooc.cn/ttsPage/login.html")
+                  img#logo-img(src="/images/pages/base/logo-tarena.png")
```

Comment:
From asana, it seems we need to show codecombat logo as well as tarena logo next to it, similar to codeninjas (except that codecombat logo should not link to /home)

Reply:
Thanks for catching this bug!